packages/app-layout/test/app-layout.test.js

```diff
@@ -305,6 +305,19 @@ describe('vaadin-app-layout', () => {
 
         expect(spy.callCount).to.be.equal(1);
       });
+
+      it('should use the drawer width as offset width', async () => {
+        const style = document.createElement('style');
+        document.documentElement.append(style);
+        const css = style.sheet;
+        css.insertRule('vaadin-app-layout::part(drawer) { width: 100px; }');
```

Comment:
nit: you can use a fixture for the style so it gets auto-cleaned after the test

```suggestion
        fixtureSync(`
          <style>
            vaadin-app-layout::part(drawer) {
              width: 100px;
            }
          </style>
        `);
```

Reply:
The `style.remove();` below also needs to be removed since the constant no longer exists